components/ai_chat/resources/untrusted_conversation_frame/components/assistant_response/tool_event.module.scss

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+
+.toolUse {
+  margin-left: 11px;
+  border-left: var(--leo-color-neutral-20) 2px solid;
+  padding: 8px 12px 8px 22px;
+  font: var(--leo-font-default-regular);
+  color: var(--leo-color-text-secondary);
+  position: relative;
+
+  .toolUseIcon {
+    --leo-icon-size: 16px;
+    --leo-progressring-size: 16px;
+    position: absolute;
+    top: calc(8px + 0.5lh - 11px); left: -12px;
```

Comment:
`lh` is new to me :laughing:

Reply:
any reason for having top/left on the same line? Makes it a bit easy to miss left

Reply:
and sorry about the hardcoded values. There's probably a better way to get this aligned right, but it's working and the icon size is fixed anyway